mlflow/utils/search_utils.py

```diff
@@ -677,47 +677,9 @@ def _parse_list_from_sql_token(cls, token):
                 error_code=INVALID_PARAMETER_VALUE,
             )
 
-    @classmethod
-    def _get_comparison_for_model_registry(cls, comparison, valid_search_keys):
```

Comment:
Why do we need to remove this method?

Reply:
The method is not used any more.

Reply:
If so, please add a comment or file a separate PR.

Reply:
Or you can file a different PR. `_get_comparison_for_model_registry` is unrelated to sqlparse's issue.

Reply:
File a new PR https://github.com/mlflow/mlflow/pull/6412 for this.